azurelinuxagent/common/osutil/default.py

```diff
@@ -1439,6 +1439,31 @@ def _get_proc_stat():
 
         return results
 
+    @staticmethod
```

Comment:
Let's not add new code under common. we want to separate the PA and GA code bases and the code dependencies are already tangled enough.